files/ru/web/http/cookies/index.html

```diff
@@ -8,31 +8,33 @@
 ---
 <p>{{HTTPSidebar}}</p>
 
-<p class="summary">HTTP cookie (web cookie, cookie браузера) - это небольшой фрагмент данных, отправляемый сервером на браузер пользователя, который тот может сохранить и отсылать обратно с новым запросом к данному серверу. Это, в частности, позволяет узнать, с одного ли браузера пришли оба запроса (например, для аутентификации пользователя). Они запоминают информацию о состоянии для протокола HTTP, который сам по себе этого делать не умеет.</p>
+<p class="summary">HTTP cookie (web cookie, куки браузера) - это небольшой фрагмент данных, отправляемый сервером и сохраняемый в браузере пользователя. Браузер отправляет куки на сервер с каждым новым запросом. Это, в частности, позволяет узнать, с одного ли браузера пришли оба запроса (например, для аутентификации пользователя). С помощью кук можно сохранить любую информацию о состоянии, HTTP-протокол сам по себе этого делать не умеет.</p>
```

Comment:
Потерялось из оригинала "browser may store the cookie and send it back":
```suggestion
<p class="summary">HTTP cookie (web cookie, куки браузера) - это небольшой фрагмент данных, который сервер отправляет браузеру пользователя. Браузер может сохранить этот фрагмент у себя и отправлять на сервер с каждым последующим запросом. Это, в частности, позволяет узнать, с одного ли браузера пришли несколько запросов (например, для аутентификации пользователя). С помощью кук можно сохранить любую информацию о состоянии, HTTP-протокол сам по себе этого делать не умеет.</p>
```

Reply:
Ещё я бы подумал над "It remembers stateful information for the stateless HTTP protocol."

Reply:
Кажется, что здесь пояснение типа "Таким образом мы можем завести состояние для HTTP-протокола, у которого состояние не предусмотрено"

Reply:
> Кажется, что здесь пояснение типа "Таким образом мы можем завести состояние для HTTP-протокола, у которого состояние не предусмотрено"

Эта мысль раскрывается в этом предложении:
> С помощью кук можно сохранить любую информацию о состоянии, HTTP-протокол сам по себе этого делать не умее